cmd/state-installer/cmd.go

```diff
@@ -419,3 +426,16 @@ func resolveInstallPath(path string) (string, error) {
 		return installation.DefaultInstallPath()
 	}
 }
+
+func assertCompatibility() error {
+	if sysinfo.OS() == sysinfo.Windows {
+		osv, err := sysinfo.OSVersion()
+		if err != nil {
+			return locale.WrapError(err, "windows_compatibility_warning", "", err.Error())
+		} else if osv.Major < 10 || osv.Micro < 17134 {
```

Comment:
Is there a risk that Windows 11.0.0.0 will break this? (I just made up that micro, but you get the idea)